### ISSUE TITLE
Similar to Tensorgram

### ISSUE THREAD

**Issue by ksdkamesh99** [Rishit-dagli/TF-Watcher#95]
Hello @Rishit-dagli @ayushmaang @shreyaparadkar @harshgeek4coder 

I have seen your project but it is almost similar to my project which is licensed. 

TensorGram: https://github.com/ksdkamesh99/TensorGram

I request you to please check this project. You can connect me via ksdkamesh99@gmail.com.

**Comment by Rishit-dagli** [comment]
Hi @ksdkamesh99 ,
Just saw your work Sai Durga Kamesh Kota, great one.

However, I don't see how these projects are anywhere near same. We have completely different goals and the only thing similar is overriding TensorFlow's callback classes which have been made open-source by TensorFlow 😂. Did you see TF Watcher?

**Comment by ksdkamesh99** [comment]
Hey @Rishit-dagli. I understand that. But the idea of monitoring the remote callbacks is the one which is similar. Just that you are showing in a website.

**Comment by Rishit-dagli** [comment]
I think I already replied to this exact same comment on your LinkedIn comment. Anyways for the sake of this issue:
Doesn't https://tensorboard.dev/ already send metrics remotely, does that mean your project copied TensorFlow's tool?

**Comment by ksdkamesh99** [comment]
Did I commented that you copied my project. I just told the similarity in both of the projects. As my project is opensource it is completely fine if you by any means use it in terms of idea. But I think giving proper credits to the idea will make things in open source community positively.

**Comment by ksdkamesh99** [comment]
> I think I already replied to this exact same comment on your LinkedIn comment. Anyways for the sake of this issue:
> Doesn't https://tensorboard.dev/ already send metrics remotely, does that mean your project copied TensorFlow's tool?

In that case are you confirming that you copied my project idea without my intervention. I don't want to make it as an issue to extend but my point is if in that case any project idea can be copied then what is the use of keeping it open source. If you are really interested in making a project you please do your search on github. Tell me one thing if some one comes and copies your current project idea and changed some components of it then won't you raise the issue.

**Comment by Rishit-dagli** [comment]
We don't use your project in any way, I saw your project just today. The idea of sending remote metrics is quite common and already implemented. I think this conversation is simply futile and you trying to forcefully find out similarities. I already showed the only thing similar in these projects in https://github.com/Rishit-dagli/TF-Watcher/issues/95#issuecomment-901558618 .

> Rishit Dagli In that case are you confirming that you copied my project idea by letting me know that I copied from tensorflow's tool.

Did you even see the comment. I am saying the idea was originated from TensorBoard.

> But I think giving proper credits to the idea will make things in open source community positively.

Definitely, **we were inspired by TensorBoard**, which has done work in a similar area, I will write credits to TensorBoard. BTW TensorBoard does allow to log metrics _remotely_ and we did build on **their** idea.

> Tell me one thing if someone comes and copies your current project idea and changed some components of it then won't you raise the issue.

Except, in this case, the project is completely different and inspiration for the idea was taken from TensorBoard.

I will end the conversation from my end here, I have already shown the inspiration for the idea and just think this is a non-sensical conversation, our project doesn't even have the same goals as yours.
